media/css/firefox/browsers/windows-64-bit.scss

```diff
@@ -51,6 +51,10 @@ $image-path: '/media/protocol/img';
             margin-bottom: $spacing-md;
         }
     }
+
+    .download-firefox {
```

Comment:
**Nit:** The button needs some horizontal margins on smaller screens

![Screenshot 2023-10-11 at 1 43 14 PM](https://github.com/mozilla/bedrock/assets/42309026/3e4a382a-0011-40fa-a03d-bf92262f8000)